adi/ad7625.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2024 Analog Devices, Inc.
+#
+# SPDX short identifier: ADIBSD
+
+from adi.context_manager import context_manager
+from adi.rx_tx import rx
+
+
+class ad7625(rx, context_manager):
+    """AD7625 device"""
+
+    _compatible_parts = [
+        "ad7625",
+        "ad7626",
+        "ad7960",
+        "ad7961",
+    ]
+
+    _device_name = ""
+    _complex_data = False
+    _rx_channel_names = ["voltage0-voltage1"]
```

Comment:
Is this channel correct? Seems odd

Reply:
Yes, it's a differential antiphase channel.